Doc/library/pprint.rst

```diff
@@ -81,29 +88,47 @@ The :mod:`pprint` module defines one class:
 
 The :mod:`pprint` module also provides several shortcut functions:
 
-.. function:: pformat(object, indent=1, width=80, depth=None, *, compact=False)
+.. function:: pformat(object, indent=1, width=80, depth=None, *, \
+                      compact=False, sort_dicts=True)
 
    Return the formatted representation of *object* as a string.  *indent*,
-   *width*, *depth* and *compact* will be passed to the :class:`PrettyPrinter`
-   constructor as formatting parameters.
+   *width*, *depth*, *compact* and *sort_dicts* will be passed to the
+   :class:`PrettyPrinter` constructor as formatting parameters.
 
    .. versionchanged:: 3.4
       Added the *compact* parameter.
 
+   .. versionchanged:: 3.8
+      Added the *sort_dicts* parameter.
+
+
+.. function:: pp(object, *args, sort_dicts=False, **kwargs)
```

Comment:
Would it be worth explicitely saying that this is an alias of pprint() with `sort_dict=False` ?

Reply:
There is already a mention of `*args` and `**kwargs` being passed to `pprint` and `sort_dicts=False` is in the signature of `pp` so I don't think it is necessary.

I would like to also like to avoid giving to much details about `pp` implementation because we may change some things if another data structure of Python changes like dicts did.

Reply:
(if we were to do change what this function does, we would also change the doc, but I'm good with that.)

Reply:
> Would it be worth explicitely saying that this is an alias of pprint() with sort_dict=False ?

At first glance I was puzzled why both `pp()` and `pprint()` existed, and it took a moment to spot the different default `sort_dict` value - and I ended up having to confirm via source code inspection that there were no other differences between the two.

As such, the above mentioned docs would have been useful for me personally :-)